### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by brownfrosamurai**
uxDesign assignment was added by meduna oluwafemi

### CODE REVIEW

**Comment by otaruMendez** [comment]
Great job! @brownfrosamurai

**Review by otaruMendez — DISMISSED** [review]

**Review by goke-epapa — COMMENTED** [review]
Well done @brownfrosamurai.

A few comments:

1. Please update the title of your Pull Request (PR) to reflect the changes you made.
For example `UX Design Task` or `Git and Version Control Task`
2. Your folder structure doesn't seem to fit the requirements, I see you have a `Git and version control` folder, then you have a folder named `meduna oluwafemi`. I believe the folder with your name should be directly in the `submissions` folder. can you please clarify?

**Comment by brownfrosamurai** [comment]
Hi Goke,

I have have made the necessary changes to my pull request and I have also
made a new pull request with my most recent assignment.

I had to upload the files using the 'drag and drop' method, I hope this is
just as acceptable.

I ran into some problems as regards using git bash to commit and add files
to my github account, Can you help me in sorting out this problem?

Regards.

**Comment by adaugochi** [comment]
Hello, you are submitting two index.html files, why? Also, your file directory should be `submissions folder->your name folder-> name of the task (e.g HTML, UxDesign etc) folder-> the file you want to submit`.

**Comment by brownfrosamurai** [comment]
Hi Maryfaith,

The first assignment had some mistakes which i corrected in the second HTML
assignment I submitted.

I have tried a few things in attempts to fix the order of the directory,
but I have not been able to do so, Can you help me understand what to do.

**Comment by adaugochi** [comment]
@brownfrosamurai can you send me a DM on slack (Adaa)

**Comment by tegaphilip** [comment]
@brownfrosamurai you should have just one html file. delete the others and push to the repo again so that the single html file can be properly reviewed.

File: meduna oluwafemi/uxDesign/HTML/css/style.css [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+    z-index: 2;
+}
+
+p {
+    text-align: center;
+    font-size: 13px;
+    color:white;
+}
+
+/* ::placeholder {
```
Comment (yemexx1) [inline meduna oluwafemi/uxDesign/HTML/css/style.css]: Please remove this comment

File: submissions/index.html [hunk 2]
```diff
@@ -0,0 +1,151 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>sign-up page</title>
+    <style>
```
Comment (yemexx1) [inline submissions/index.html]: Could you please extract this into a css file?

**Review by yemexx1 — DISMISSED** [review]

File: meduna oluwafemi/uxDesign/HTML/css/style.css (hunk 1 above)
Comment (brownfrosamurai) [reply meduna oluwafemi/uxDesign/HTML/css/style.css]: I have made the necessary changes.

**Comment by tegaphilip** [comment]
@brownfrosamurai well done. However, now you have two css files. please delete one of them just like you did when you had multiple html files. Make sure the one you deleted is not the one your html code is pointing to. I think your html file is pointing to `meduna oluwafemi/uxDesign/HTML/css/style.css` so you should delete the other one which is `submissions/css/style.css`. 
Or is there a reason why you have two css files? Then push your code again.

Also update the title of your PR as suggested here. https://github.com/Dufuna-CodeCamp/Dufuna-CodeCamp19/pull/18#pullrequestreview-306898227

**Review by tegaphilip — DISMISSED** [review]

**Review by JohnItoo — COMMENTED** [review]
@brownfrosamurai  This is a reminder to attend to the comment on your `style.css` file. Please kindly remove the commented out block. Thanks.

**Review by tegaphilip — APPROVED** [review]

**Review by JohnItoo — APPROVED** [review]

**Pull request merged**